### DISCUSSION TITLE
里士满北边的有钱人 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#442] [General]
# 里士满北边的有钱人 - Miao Yu | 于淼

https://yufree.cn/cn/2023/09/03/rich-men-north-of-richmond/

**Comment by ordiy** [comment]
前10年生活在农村，最近20年生活城市，经历了农村、小城镇、城市、大都市的生活，在城市生活的时候总有一种割裂感，所以开始向往“诗和远方的田野”。但是生活还要继续在大都市，就如同（Rich Men North Of Richmond）中的“selling my soul“， 这种割裂感的矛盾愈发激烈但是又没找到深层次的原因，看了作者的博客，我意识到了这是自己思维（or思想观念）的问题，我的脑子没转过来（-_-）,据如同作者说的“深的问题可能就在于我们这个从狩猎采集社会后就没怎么进化过的大脑，总想通过回到过去去解决未来的问题是既蠢又坏的“。  从理性和事实来看，我“需要深刻的理解和适应时代”。
 生活在这个丛里里的猴子🐒 ，我想理解这个钢铁丛林！
  我把作者的观点整理一张图，有的地方不太准确。

![](https://raw.githubusercontent.com/ordiy/study_notes/master/res/image/node_image/blog_20231129114033.png)

**Reply by yufree** [reply]
我只能说你的总结比我啰嗦那些清晰多了，图非常棒！
